libraries/Microsoft.Bot.Builder.LUIS/LuisModel.cs

```diff
@@ -22,11 +22,11 @@ public LuisModel(string modelId, string subscriptionKey, Uri uriBase, LuisApiVer
             ApiVersion = apiVersion;
         }
 
-        public string ModelID { get; }
+        public string ModelID { get; set; }
```

Comment:
There are no unit tests around this entire LuisModel class. 

Please add.

Reply:
Done